doc/kuksa_analysis/kuksa_design_topics.md

```diff
@@ -0,0 +1,725 @@
+# Design Topics
```

Comment:
We have some open topics in this document, like questions and statements started with `? `

Shall we possibly add a short into to this document saying that this document tries to collect current design decisions as well as topics where we not yet have made any decision.